tests/Jellyfin.Naming.Tests/TV/EpisodePathParserTest.cs

```diff
@@ -7,43 +7,94 @@ namespace Jellyfin.Naming.Tests.TV
         // TODO: [InlineData("/After Life 1x06 Episodio 6 [WEB-DL NF 1080p h264 Dual DD 5.1 Sub].mkv", "After Life", 1, 6)]
         // TODO: [InlineData("/Season 4/Uchuu.Senkan.Yamato.2199.E03.avi", "Uchuu Senkan Yamoto 2199", 4, 3)]
         // TODO: [InlineData("The Daily Show/The Daily Show 25x22 - [WEBDL-720p][AAC 2.0][x264] Noah Baumbach-TBS.mkv", "The Daily Show", 25, 22)]
         // TODO: [InlineData("Watchmen (2019)/Watchmen 1x03 [WEBDL-720p][EAC3 5.1][h264][-TBS] - She Was Killed by Space Junk.mkv", "Watchmen (2019)", 1, 3)]
         // TODO: [InlineData("/The.Legend.of.Condor.Heroes.2017.V2.web-dl.1080p.h264.aac-hdctv/The.Legend.of.Condor.Heroes.2017.E07.V2.web-dl.1080p.h264.aac-hdctv.mkv", "The Legend of Condor Heroes 2017", 1, 7)]
-        public void ParseEpisodesCorrectly(string path, string name, int season, int episode)
+        public void ParseEpisodesCorrectly(string path, bool isDirectory, string name, int season, int episode)
         {
             NamingOptions o = new NamingOptions();
             EpisodePathParser p = new EpisodePathParser(o);
-            var res = p.Parse(path, false);
+            var res = p.Parse(path, isDirectory);
 
             Assert.True(res.Success);
             Assert.Equal(name, res.SeriesName);
             Assert.Equal(season, res.SeasonNumber);
             Assert.Equal(episode, res.EpisodeNumber);
         }
+
+        [Theory]
+        [InlineData("/test/01-03.avi", true, true)]
+        public void EpisodePathParserTest_DifferentExpressionsParameters(string path, bool? isNamed, bool? isOptimistic)
+        {
+            NamingOptions o = new NamingOptions();
+            EpisodePathParser p = new EpisodePathParser(o);
+            var res = p.Parse(path, false, isNamed, isOptimistic);
+
+            Assert.True(res.Success);
+        }
+
+        [Fact]
+        public void EpisodePathParserTest_FalsePositivePixelRate()
+        {
+            NamingOptions o = new NamingOptions();
+            EpisodePathParser p = new EpisodePathParser(o);
+            var res = p.Parse("Series Special (1920x1080).mkv", false);
+
+            Assert.False(res.Success);
+        }
+
+        [Fact]
+        public void EpisodeResolverTest_WrongExtension()
+        {
+            var res = new EpisodeResolver(new NamingOptions()).Resolve("test.mp3", false);
+            Assert.Null(res);
+        }
+
+        [Fact]
+        public void EpisodeResolverTest_WrongExtensionStub()
+        {
+            var res = new EpisodeResolver(new NamingOptions()).Resolve("dvd.disc", false);
+            Assert.NotNull(res);
+            Assert.True(res!.IsStub);
+        }
+
+        [Fact]
+        public void EpisodePathParserTest_EmptyDateParsers()
+        {
+            NamingOptions o = new NamingOptions()
+            {
+                EpisodeExpressions = new[] { new EpisodeExpression("(([0-9]{4})-([0-9]{2})-([0-9]{2}) [0-9]{2}:[0-9]{2}:[0-9]{2})", true) }
```

Comment:
Where does this expression come from?

Reply:
It is basically a dummy test to keep code coverage while keeping a few useful lines of codes that are not currently used. I have added a comment to it. 
The code involved is in EpisodePathParser.cs:130 and for me, the code there makes sense and could be reached with additional regex in the future, but maybe it would make more sense to comment it out?